firebase-abt/src/main/java/com/google/firebase/abt/FirebaseABTesting.java

```diff
@@ -162,6 +166,70 @@ public void removeAllExperiments() throws AbtException {
     return experimentInfos;
   }
 
+  /**
+   * Sets an experiment to be active in GA metrics reporting by setting a null triggering condition
+   * on the provided experiment. This results in the experiment being active as if it was triggered
+   * by the triggering condition event being seen in GA.
+   *
+   * <p>Note: This is a blocking call and therefore should be called from a worker thread.
+   *
+   * @param activeExperiment The {@link AbtExperimentInfo} that should be set as active in GA.
+   * @throws AbtException If there is no Analytics SDK.
+   */
+  @WorkerThread
+  public void reportActiveExperiment(AbtExperimentInfo activeExperiment) throws AbtException {
+    throwAbtExceptionIfAnalyticsIsNull();
+    ArrayList<AbtExperimentInfo> activeExperimentList = new ArrayList<>();
+
+    // Remove trigger event if it exists, this sets the experiment to active.
+    Map<String, String> activeExperimentMap = activeExperiment.toStringMap();
+    activeExperimentMap.remove(AbtExperimentInfo.TRIGGER_EVENT_KEY);
+
+    // Add experiment to GA
+    activeExperimentList.add(AbtExperimentInfo.fromMap(activeExperimentMap));
+    addExperiments(activeExperimentList);
+  }
+
+  /**
+   * Adds an experiment to be active in GA by setting a null triggering condition on the provided
+   * experiment. This results in the experiment being active as if it was triggered by the
+   * triggering condition event being seen in GA.
+   *
+   * <p>Note: This is a blocking call and therefore should be called from a worker thread.
+   *
+   * @param activeExperiment The {@link FirebaseAbt.ExperimentPayload} that should be set as active
+   *     in GA.
+   * @throws AbtException If there is no Analytics SDK.
+   */
+  @WorkerThread
+  public void reportActiveExperiment(FirebaseAbt.ExperimentPayload experimentPayload)
+      throws AbtException {
+    reportActiveExperiment(AbtExperimentInfo.fromExperimentPayload(experimentPayload));
+  }
+
+  /**
+   * Cleans up all experiments which are active in GA but not currently running. This method is
+   * meant to be used to ensure all running experiments should indeed be running.
+   *
+   * <p>Note: This is a blocking call and therefore should be called from a worker thread.
+   *
+   * @param runningExperiments the currently running {@link AbtExperimentInfo}s, any active
+   *     experiment that is not in this list will be removed from GA reporting.
+   * @throws AbtException If there is no Analytics SDK.
+   */
+  @WorkerThread
+  public void reportRunningExperiments(List<AbtExperimentInfo> runningExperiments)
```

Comment:
consider replacing "report" with a different word like "check/validate"? Since this function is used to remove obsolete experiments from GA

Reply:
yeah validate seems good actually, I'll switch to that